packages/wonder-stuff-sentry/src/__tests__/kind-sentry-error.test.js

```diff
@@ -0,0 +1,417 @@
+// @flow
+import {KindError, Errors} from "@khanacademy/wonder-stuff-core";
```

Comment:
Will need the github action to build locally, I think (and maybe call `lerna bootstrap` if we want this to pass before any package is published.

Reply:
wonder-blocks uses `import/no-unresolved` as well.  `yarn test` is set up to build things first before it runs lint.  You may also have to do `lerna bootstrap` as well as you mentioned.